### DISCUSSION TITLE
Raspberry Pi Read-Only

### DISCUSSION THREAD

**Discussion by timostubler** [kizniche/Mycodo#1371] [General]
Hello,

what is the best way to use mycodo with the SD-card of the raspberry pi set to read-only?
I did this, since my SD card got corrupted multiple times, and I want to avoid writing on the SD-card at all.
Any hints on configuration?

Best regards,
Timo

**Comment by H20frks** [comment]
If you are using the microSD I would move to a SSD. 
https://thepihut.com/blogs/raspberry-pi-tutorials/how-to-set-up-an-ssd-with-the-raspberry-pi
